### PR TITLE
Adding Roman to the company page

### PR DESCRIPTION
**Pull request by codebryo**
Figured I am not on the team page yet, so I fixed that. 
Was not able to render a preview of the app though but I guess it works, unless I missed anything.

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/6066/commits/dabee9b386a6bc7a799f193f3d6d4bc11c097b1d failed.

File: packages/homepage/src/components/TeamMember.js [hunk 1]
```diff
@@ -129,6 +129,12 @@ const people = [
     color: '#76D0FB',
     team: 'Operations',
   },
+  {
+    name: 'Roman Kuba',
+    pic: roman,
```
Comment (danilowoz) [inline packages/homepage/src/components/TeamMember.js:134]: Missing the image importing

Comment (codebryo) [reply packages/homepage/src/components/TeamMember.js:135]: Yeah, just found it. Thanks 🙌

**Review by danilowoz — APPROVED** [review]
Welcome to the team 🎉

**Pull request merged**